### PR TITLE
Integrate wiki into README.md and files in the docs directory

### PR DESCRIPTION
**Pull request by remyers**
The wiki on [github](https://github.com/ACINQ/eclair/wiki) has become outdated in a few places relative to similar information in README.md, in particular Installation and Configuration. Also few people know about the wiki, even though we did have a few links to it in other documentation files.

To make sure the information is up-to-date I have also tried to replicate the steps in the wiki pages that have been transformed in to files in the docs directory.

### CODE REVIEW

**Comment by remyers** [comment]
Should we link to the more user friendly [Trampoline Proposal](https://github.com/lightning/bolts/blob/trampoline-routing-no-gossip/proposals/trampoline.md) instead of the [PR](https://github.com/lightning/bolts/pull/829) from the TrampolinePayments.md guide?

**Comment by t-bast** [comment]
I think the PR makes more sense (the one you linked does contain the friendly proposal document)

**Comment by remyers** [comment]
I've only closely reviewed and fixed up the new files migrated from the Wiki to make sure they are still correct:
- [X]  docs/API.md
- [X] docs/FAQ.md
- [X] docs/Features.md
- [X] docs/Guides.md
- [X] docs/MultipartPayments.md
- [X] docs/TrampolinePayments.md
- [X] docs/Usage.md

File: README.md [hunk 1]
```diff
@@ -177,13 +178,13 @@ eclair-node-<version>-<commit_id>/bin/eclair-node.sh -Dlogback.configurationFile
 ### Backup
 
 You need to backup:
-- your bitcoin core wallet
-- your eclair channels
+- your Bitcoin core wallet
```
Comment (t-bast) [inline README.md:181]: nit: maybe also capitalize "Core"? Same in a few other places

```suggestion
- your Bitcoin Core wallet
```

File: docs/FAQ.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# FAQ
+
+## What does it mean for a channel to be "enabled" or "disabled" ?
+
+A channel is disabled if a `channel_update` message has been broadcast for that channel with the `disable` bit set (see [BOLT 7](https://github.com/lightning/bolts/blob/master/07-routing-gossip.md#the-channel_update-message)). It means that it the channel still exists but cannot be used to route payments, until it has been re-enabled.
```
Comment (t-bast) [inline docs/FAQ.md:5]:
```suggestion
A channel is disabled if a `channel_update` message has been broadcast for that channel with the `disable` bit set (see [BOLT 7](https://github.com/lightning/bolts/blob/master/07-routing-gossip.md#the-channel_update-message)). It means that the channel still exists but cannot be used to route payments, until it has been re-enabled.
```

**Review by t-bast — COMMENTED** [review]
Thanks, looks good to me, only a couple minor comments.

File: README.md (hunk 1 above)
Comment (remyers) [reply README.md:181]: Good point; I'll go through and make sure Core is consistently capitalized.

Comment (remyers) [reply README.md]: Seems this was the only place we needed to s/core/Core

Comment (t-bast) [reply README.md]: Are you sure? I see another one two lines below ;)

Comment (remyers) [reply README.md]: Indeed! ha, not sure how I missed that. Anyway, fixed last commit.

**Review by t-bast — APPROVED** [review]
LGTM, thanks!

Make sure to select the "squash and merge" option when merging that PR, which will also let you clean-up the commit message.

**Comment by remyers** [comment]
@t-bast do we want to hide the wiki now in Github?

**Comment by t-bast** [comment]
Yes good idea, I'll do that

**Pull request merged**